src/desktop/apps/artwork/components/partner/index.jade

```diff
@@ -1,44 +1 @@
-if sd.ENABLE_NEW_BUY_NOW_FLOW || (user && user.hasLabFeature('New Buy Now Flow'))
-  //- Hide old Partner / Gallery info section in favor of ArtworkDetails
-else
-  if helpers.partner.isDisplayable(artwork.partner)
-    - var partner = artwork.partner
-    - var profile = partner.profile
-
-    section.artwork-section.artwork-partner( class='js-artwork-partner' )
-      .artwork-partner__icon: a(
-        href= partner.href
-        data-initials= partner.initials
-        class= (profile && profile.icon ? 'artwork-partner__icon--icon' : undefined)
-        style= (profile && profile.icon ? 'background-image: url(#{profile.icon.url})' : undefined)
-      )
-
-      .artwork-partner__metadata
-        a.artwork-partner__metadata__name( href= partner.href )
-          = partner.name
-
-        if partner.locations && partner.locations.length
-          .artwork-partner__metadata__locations
-            = helpers.partner.cities(partner.locations)
-
-        if profile
-          .artwork-partner__metadata__bio
-            = profile.bio
-
-        .artwork-partner__metadata__actions
-          //- TODO BNMO: Update to hide if the artwork is acquireable
-          if artwork.is_inquireable
-            a.avant-garde-button-black.is-small(
-              class='js-artwork-partner-contact analytics-artwork-contact-seller'
-              data-artwork_id= artwork._id
-              data-context_type='partner-module'
-            )
-              | Contact #{artwork.partner.type}
-
-          if profile
-            a.avant-garde-button-white.follow-button.is-small(
-              class='js-artwork-partner-follow'
-              data-type='profile'
-              data-_id= profile._id
-              data-id= profile.id
-            )
+//- Hide old Partner / Gallery info section in favor of ArtworkDetails
```

Comment:
Looks odd, why include an empty Jade file with just a comment? I think it was there in the `if` statement so that you could still render other stuff in the non BNMO case (I'm getting fuzzy on all these cases/contexts), but with BNMO its likely this whole file(component?) can go.

Reply:
Oh yeah, you can actually go really far down this rabbit hole deleting components etc. but since the artwork page is in the middle of being redone it didn't seem worth it.

Reply:
I think this whole file can go for every case. We are displaying ArtworkDetails always now and it includes Partner. So no need to keep this partner around.

Reply:
I can remove now but hopefully in a short while this whole app will go anyways?